benches/enrichment_tables_file.rs

```diff
@@ -0,0 +1,123 @@
+use criterion::{criterion_group, criterion_main, BatchSize, Criterion};
+use std::collections::BTreeMap;
+use vector::enrichment_tables::{file::File, Condition, Table};
+
+criterion_group!(
+    name = benches;
+    config = Criterion::default().noise_threshold(0.02).sample_size(10);
```

Comment:
I imagine you could drop the `sample_size()` now. I'm guessing you added it when the times were much slower.